app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager.rb

```diff
@@ -8,6 +8,7 @@ class ManageIQ::Providers::IbmCloud::VPC::CloudManager < ManageIQ::Providers::Cl
   require_nested :Template
   require_nested :Vm
   require_nested :LoggingMixin
+  require_nested :EventCatcher
```

Comment:
alphabetical? (not sure if this was ordered at the end on purpose or not)

Reply:
It was not on purpose, thanks I will order the other `require_nested` statements in this file as well while I'm at it